README.md

```diff
@@ -88,6 +88,9 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
+
```

Comment:
Minor: Could you remove this newline (elsewhere too) for consistency? Thanks! :)